### PR TITLE
Separate login-as banner for small screens

### PR DESCRIPTION
**Pull request by esoergel**
## Product Description

Make "login as" banner display separately on small screens.  Followup from https://github.com/dimagi/commcare-hq/pull/34930, which made it look like this:

image

And after this PR:

image

And in live preview:

image

## Technical Summary
https://dimagi.atlassian.net/browse/USH-4862

## Safety Assurance

I've tested this locally and on staging.

### Rollback instructions
- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss [hunk 1]
```diff
@@ -176,15 +176,8 @@ button.clickable-icon {
...
-  background-color: $cc-light-warm-accent-hi;
-  color: black;
-  padding-top: 1rem;
-  padding-bottom: 1rem;
-  border: none;
-  margin-top: -1px;
-  text-align: center;
-  font-size: 14px;
+.restore-as-banner {
+  font-size: 11px;
```
Comment (MartinRiese) [inline corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss:180]: I wonder if we should just call it `font-size-11` similar to the color classes available.

**Review by MartinRiese — APPROVED** [review]
+1 for a bunch of deleted code.

File: corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss (hunk 1 above)
Comment (esoergel) [reply corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/module.scss:180]: I thought about that - Bootstrap has [font size utilities](https://getbootstrap.com/docs/5.0/utilities/text/#font-size), but they only go down as small as normal sized text, which is too big here.  We could extend that same system a couple clicks longer

File: corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/views.js [hunk 2]
```diff
@@ -36,12 +35,12 @@ hqDefine("cloudcare/js/formplayer/users/views", [
...
-                const templateId = (usersModels.getCurrentUser().isAppPreview ?
-                                    "#app-preview-restore-as-template" :
-                                    "#webapps-restore-as-template");
+                const templateId = (this.options.smallScreen || usersModels.getCurrentUser().isAppPreview ?
+                                    "#restore-as-banner-template" :
+                                    "#restore-as-pill-template");
                 return _.template($(templateId).html() || "");
             } else {
-                return "";
+                return _.template("");
```
[the remark this answers is not in the document]
Comment (esoergel) [reply corehq/apps/cloudcare/static/cloudcare/js/formplayer/users/views.js:43]: Yeah, I broke this in a recent PR, but didn't notice it until here.

**Review by Robert-Costello — APPROVED** [review]

**Pull request merged**